Tests/HubTests/HubApiTests.swift

```diff
@@ -87,6 +87,63 @@ class HubApiTests: XCTestCase {
             XCTFail("\(error)")
         }
     }
+    
+    func testGetFileMetadata() async throws {
+        do {
+            let url = URL(string: "https://huggingface.co/coreml-projects/Llama-2-7b-chat-coreml/resolve/main/config.json")
+            let metadata = try await Hub.getFileMetadata(fileURL: url!)
+            
+            XCTAssertNotNil(metadata.commitHash)
+            XCTAssertNotNil(metadata.etag)
+            XCTAssertEqual(metadata.location, url?.absoluteString)
+            XCTAssertEqual(metadata.size, 163)
+        } catch {
+            XCTFail("\(error)")
+        }
+    }
+    
+    func testGetFileMetadataBlobPath() async throws {
+        do {
+            let url = URL(string: "https://huggingface.co/coreml-projects/Llama-2-7b-chat-coreml/blob/main/config.json")
+            let metadata = try await Hub.getFileMetadata(fileURL: url!)
+            
+            XCTAssertEqual(metadata.commitHash, nil)
+            XCTAssertTrue(metadata.etag != nil && metadata.etag!.hasPrefix("1081d-"))
+            XCTAssertEqual(metadata.location, url?.absoluteString)
+            XCTAssertEqual(metadata.size, 67613)
```

Comment:
```suggestion
            XCTAssertTrue(metadata.etag != nil && metadata.etag!.hasPrefix("10841-"))
            XCTAssertEqual(metadata.location, url?.absoluteString)
            XCTAssertEqual(metadata.size, 67649)
```

Any chance you were testing a different file? I got different values which I verified with the Python library.